modules/config/pages/eap-config.adoc

```diff
@@ -1,11 +1,159 @@
 = JBoss EAP Configuration
 
-Brief blurb about the different ways to configure JBoss EAP....
+== Overview of Standalone Server
 
-== Section Header
+* Standalone servers are perfect for running a single instance of Red Hat JBoss EAP.
```

Comment:
Add a bullet - you can run multiple standalone instances on the same server or across different servers. Each server is managed "standalone", that is, individually.